claragenomics/io/bedio.py

```diff
@@ -14,11 +14,12 @@
 import pandas as pd
 
 
-def read_intervals(bed_file):
-    """Function to read genomic intervals from a BED file into a DataFrame.
+def read_intervals(bed_file, skip=0):
```

Comment:
Why do we need skip option ?

Reply:
In case we're reading an interval file with a header. Currently used by `peak2bw.py`.